client/src/pages/Landing.jsx

```diff
@@ -206,12 +173,11 @@ class Landing extends Component {
                             One of Austin’s most prized attractions, the Barton Creek Greenbelt is filled
                             with thrilling biking trails, pristine swimming holes, and beautiful limestone 
                             bluffs that provide for excellent rock climbing. Unfortunately for newcomers, accessing 
-                            this treasure can be a little confusing. To help,
-                            here is a complete guide to all the public access points for Austin’s natural gem. To begin, hit 
-                            continue and click on any access point on the map for a detailed explanation. 
+                            this local treasure can be a little confusing. To help, here is a complete guide to all the public 
+                            access points for Austin’s natural gem. Click CONTINUE to begin.  
                             <br/><br/>
-                            Don't forget to also log-in and check out current trail updates from fellow Greenbelters or to post your 
-                            own access point status updates!
+                             *Don't forget you can also LOGIN and check out each location's updates from fellow Greenbelters 
```

Comment:
Love it! Great additional instructions here

Reply:
Nice!